packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/utils/getDisplayedFilters.js

```diff
@@ -0,0 +1,60 @@
+import ComboboxFilter from '../Filters/ComboboxFilter';
+
+const customOperators = [
+  {
+    intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$eq', defaultMessage: 'is' },
+    value: '$eq',
+  },
+  {
+    intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$ne', defaultMessage: 'is not' },
+    value: '$ne',
+  },
+];
+
+const getDisplayedFilters = ({ actionOptions, userOptions }) => {
+  return [
+    {
+      name: 'action',
+      metadatas: {
+        label: 'Action',
+        options: actionOptions,
+        customOperators,
+        customInput: ComboboxFilter,
+      },
+      fieldSchema: { type: 'enumeration' },
+    },
+    {
+      name: 'user',
+      metadatas: {
+        label: 'User',
+        options: userOptions,
+        customOperators: [
+          ...customOperators,
+          {
+            intlLabel: {
+              id: 'components.FilterOptions.FILTER_TYPES.$null',
+              defaultMessage: 'is null',
+            },
+            value: '$null',
+          },
+          {
+            intlLabel: {
+              id: 'components.FilterOptions.FILTER_TYPES.$notNull',
+              defaultMessage: 'is not null',
+            },
+            value: '$notNull',
+          },
+        ],
+        customInput: ComboboxFilter,
+      },
+      fieldSchema: { type: 'relation', mainField: { name: 'id', schema: { type: 'integer' } } },
```

Comment:
Does that mean the ID is an integer? I thought in other places Strapi expected a string for the ID

Reply:
It was more about using the type provided here to get the appropriate operator options in [this file](https://github.com/strapi/strapi/blob/96e09dbb938ea1c33e9f7095efb64784baa22dd4/packages/core/helper-plugin/lib/src/components/FilterPopoverURLQuery/utils/getFilterList.js#L7)

Since I'm passing custom operators now I just removed the type and made schema optional for the condition `schema?.type`

https://github.com/strapi/strapi/blob/96e09dbb938ea1c33e9f7095efb64784baa22dd4/packages/core/helper-plugin/lib/src/components/FilterPopoverURLQuery/utils/getFilterList.js#L7